src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -81,4 +77,20 @@ public SectionResponse saveSection(Long lineId, LineRequest request) {
                 .map(SectionResponse::listOf)
                 .orElseThrow(() -> new LineNotFoundException(lineId));
     }
+
+    public void deleteSectionByStationId(Long lineId, Long stationId) {
```

Comment:
> 구현과 관련된 모든 정보를 캡슐화하고 객체의 퍼블릭 인터페이스에는 협력과 관련된 의도만을 표현해야 한다.

개인적으로 퍼블릭 메서드 명을 정할때는 협력에 맞게 추상적인 작성하는 것을 선호하는데요. 
그 이유는 내부 구현이 바뀌더라도, 메서드 명을 유지할 수 있기 때문입니다.
메서드 명을 유지한다는 것은 생각보다 단순해보이기도하지만, 처음 정의한 협력을 바꾸지 않는다는 관점에서 바라보면 굉장히 중요하게 느껴지기도 하네요 🤔

제가 말씀 드린 방법 또한 정답은 아니며 구체적으로 메서드명을 가져가는 것 또한 직관적이니 장점이 있다고는 생각합니다 !
창호님은 어떻게 생각하시는지 의견 들려주시면 감사합니다 🙏 

제가 이야기드린 대로 바꿔보자면 조금 과할지 도모르지만 아래와 같은 방법도 있을 것 같습니다 :) 
바꾸지 않으셔도 괜찮습니다. 참고만 해주세요 😄 
```suggestion
    public void deleteSection(Long lineId, SectionDeleteReqeust request) {
```


>

Reply:
전부 맞는 말씀입니다!
내부 구현은 숨기도록 메소드명을 고민하여 작성하는 연습을 계속 해보겠습니다.~!
👍 🙇